components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/LifeStatusController.java

```diff
@@ -58,9 +58,7 @@
 
     private static final String DECEASED = "deceased";
 
-    private static final String UNBORN = "unborn";
```

Comment:
I recall we explicitly decided to add this on in the past (I think the logic was that a proband may be an unborn child with some detected phenotypes or genotypes, and that when parents come for help that would be the patient/proband)

Lets confirm with @marta- and Felicia that we don't want it any more (even if it was not supported in the UI before maybe it is a good time to add it)